### PR TITLE
Add lookout application form

### PR DESCRIPTION
**Pull request by burgertime-c4**

### CODE REVIEW

File: src/styles/pages/applications/lookout-application.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      border-bottom: none;
+    }
+  }
+  textarea {
+    min-height: 12rem;
+  }
+  .form__submit-button-holder {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (captainmangoC4) [inline src/styles/pages/applications/lookout-application.scss:36]:
```suggestion
    align-items: center;
    flex-direction: column;
```

File: src/pages/lookout-application.js [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+      setErrorMessage(error);
+    }
+  };
+
+  const handleSubmit = () => {
+    if (
+      !isInitialized ||
+      !user ||
+      !currentUser.isLoggedIn ||
+      !Object.values(state).every((val) => !!val)
```
[the remark this answers is not in the document]
Comment (captainmangoC4) [reply src/pages/lookout-application.js]: [EDIT] Fixed this in https://github.com/code-423n4/code423n4.com/pull/8073/commits/13a8dffaa6301ed39bfb7aed5242c18a10a3889a

File: src/pages/lookout-application.js [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+      helpText:
+        "Describe how the above finding demonstrates your depth of knowledge",
+      widget: "textarea",
+      required: true,
+    },
+    {
+      name: "link3",
+      label: "Third Code4rena finding link",
+      helpText:
+        "Link to a valid, high-severity finding you submitted to a Code4rena contest",
```
[the remark this answers is not in the document]
Comment (captainmangoC4) [reply src/pages/lookout-application.js:79]: Remove other references to "high severity"

File: netlify/functions/apply-for-lookout.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+    ## High severity Code4rena submission 1
+    ${link1}
+    ${details1}
+
+    ## High severity Code4rena submission 2
+    ${link2}
+    ${details2}
+
+    ## High severity Code4rena submission 3
```
[the remark this answers is not in the document]
Comment (captainmangoC4) [reply netlify/functions/apply-for-lookout.js:129]: Remove other references to "high severity"

File: src/pages/lookout-application.js [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+        "(e.g. instructor, teaching assistant, team manager, etc.) If so, please tell us a little about it.",
+      widget: "textarea",
+      required: false,
+    },
+    {
+      name: "questions",
+      label: "What questions do you have about the Lookout Guidelines?",
+      helpText: (
+        <a
+          href="https://www.notion.so/Pre-sort-Lookout-Guidelines-bead139f7ec9469197c7d46f759d89d5"
```
Comment (CloudEllie) [inline src/pages/lookout-application.js:33]: Can we change this URL? I just realized that that URL isn't public facing, but this one is: 

https://code4rena.notion.site/Pre-sort-Lookout-Guidelines-bead139f7ec9469197c7d46f759d89d5

**Review by captainmangoC4 — APPROVED** [review]

**Pull request merged**